r/src/symbols.cpp

```diff
@@ -60,11 +60,11 @@ SEXP data::classes_tbl_df =
     precious(cpp11::writable::strings({"tbl_df", "tbl", "data.frame"}));
 
 SEXP data::classes_arrow_binary =
-    precious(cpp11::writable::strings({"arrow_binary", "vctrs_vctr", "list"}));
-SEXP data::classes_arrow_large_binary =
-    precious(cpp11::writable::strings({"arrow_large_binary", "vctrs_vctr", "list"}));
-SEXP data::classes_arrow_fixed_size_binary =
-    precious(cpp11::writable::strings({"arrow_fixed_size_binary", "vctrs_vctr", "list"}));
+    precious(cpp11::writable::strings({"arrow_binary", "blob", "vctrs_vctr", "list"}));
+SEXP data::classes_arrow_large_binary = precious(
+    cpp11::writable::strings({"arrow_large_binary", "blob", "vctrs_vctr", "list"}));
+SEXP data::classes_arrow_fixed_size_binary = precious(
+    cpp11::writable::strings({"arrow_fixed_size_binary", "blob", "vctrs_vctr", "list"}));
```

Comment:
Do we also need to add `vctrs_list_of` here to match the blob classes?

```
> blob::as_blob("foo")
<blob[1]>
[1] blob[3 B]
> bar <- blob::as_blob("foo")
> class(bar)
[1] "blob"          "vctrs_list_of" "vctrs_vctr"    "list"       
```

https://github.com/tidyverse/blob/blob/bde7f8cd6495a538ef5e08fdfac6622bb9d54aa0/R/blob.R#L6

Reply:
Good catch, thanks. I will update.